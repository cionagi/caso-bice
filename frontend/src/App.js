import React from 'react';

const App = () => {
  return (
    <>
      <nav className="navbar navbar-light bg-light">
        <a className="navbar-brand" href="/">
          Caso BICE
        </a>
      </nav>
      <div className="container">
        <h1 className="my-3">Indicadores diarios FECHAAA</h1>
      </div>
    </>
  );
};

export default App;
