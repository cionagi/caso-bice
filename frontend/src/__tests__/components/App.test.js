import React from 'react';
import { render } from '@testing-library/react';

import App from '../../App';

test('Link renders correctly', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
