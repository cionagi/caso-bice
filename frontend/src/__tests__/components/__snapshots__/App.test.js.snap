// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<DocumentFragment>
  <nav
    class="navbar  custom-nav"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      Caso BICE 
    </a>
  </nav>
  <div
    class="container"
  >
    <h1
      class="my-3"
    >
      Indicadores diarios
      <span
        class="badge badge-custom ml-3"
      >
        25-04-2020
      </span>
    </h1>
    <div
      class="d-flex justify-content-center align-items-center"
      data-testid="loading-component"
    >
      <div
        class="spinner-border"
        role="status"
        style="width: 5rem; height: 5rem;"
      >
        <span
          class="sr-only"
        >
          Loading...
        </span>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
